### ISSUE TITLE
Miss accessibility on required asterisk

### ISSUE THREAD

**Issue by masseuro** [chakra-ui/chakra-ui#6870]
### Description

In formControl with isRequired  you should have this (sample from documentation)
![image](https://user-images.githubusercontent.com/131135/197157121-65bed4f3-94fb-4c52-af08-1928b3ad42a8.png)

### Problem Statement/Justification

Problem with screen reader this asterisk is not understand because of the use or role ="presentation" and aria-hidden="true"

Actual hmtl code generated
`<span role="presentation" aria-hidden="true" class="chakra-form__required-indicator css-1ssjhh">*</span>`

### Proposed Solution or API

It will be more undestanding like this,

`<span class="chakra-form__required-indicator css-1ssjhh">*</span>`

### Alternatives

Another alternatives :

``<abbr title="Required field" class="chakra-form__required-indicator css-1ssjhh">*</abbr>``

or keep <span> but add a special text for accessibility upper the form like this "All fields with * are required"

**Comment by segunadebayo** [comment]
Hi @masseuro,

Ideally, this additional accessibility requirement should be handled in your application. We add the `required` attribute to the input element, which is the native way of ensuring an input element is required.

You can change the required indicator like so:

```jsx
<FormLabel requiredIndicator={<abbr title="required field">*</abbr>} />
```

I hope this helps.
